src/lib/components/collections/collections-list/import-curl/ImportCurl.viewModel.ts

```diff
@@ -2,10 +2,4 @@ import { CollectionService } from "$lib/services/collection.service";
 
 export class ImportCurlViewModel {
   private collectionService = new CollectionService();
-
-  public importCollectionFromCurl = async (curl: string) => {
-    const response =
-      await this.collectionService.importCollectionFromCurl(curl);
-    return response;
-  };
 }
```

Comment:
If we have removed all methods from this file, and it doesn't serve any purpose so we can delete this file as well.